### PR TITLE
added </option> as an optional ommited end tag

### PR DESCRIPTION
**Pull request by pollei**
* added </option> as an optional ommited end tag
      as per http://www.w3.org/TR/html5/syntax.html#optional-tags
* updated README to deal with some link-rot

	modified:   external/slowparse/README.md
	modified:   external/slowparse/slowparse.js

  https://www.khanacademy.org/profile/spollei/

### CODE REVIEW

File: external/slowparse/slowparse.js [hunk 1]
```diff
@@ -1301,7 +1301,17 @@
...
+      "li": ["li"],
+      "rb": ["rb","rt","rtc","rp"],
+      "rt": ["rb","rt","rtc","rp"],
+      "rtc": ["rb","rtc","rp"],
+      "rp": ["rb","rt","rtc","rp"],
+      "optgroup": ["optgroup"],
+      "option": ["option"],
+      "thead": ["tbody", "tfoot"],
+      "tbody": ["tbody", "tfoot"],
+      "tfoot": ["tbody"]
```
Comment (kevinbarabash) [inline external/slowparse/slowparse.js]: Shouldn't "tr", "rb", "rt", "rtc", "rp", "optgroup", "thead", "tbody" and "tfoot" also be added to `ombittableCloeTagHtmlElements`?

**Comment by kevinbarabash** [comment]
Thank you for the pull request.  Could you add automated tests for these changes?

**Comment by pollei** [comment]
https://github.com/mozilla/slowparse/pull/81 yes there should be more things added to omittableCloseTagHtmlElements . I also have some test cases in the upstream project.
Hopefully next weekend I'll have more to show.

**Comment by kevinbarabash** [comment]
@pollei happy to hear your look at pushing your changes upstream.  I'm looking how to make it easier for live-editor to track changes to mozilla/slowparse.

**Comment by kevinbarabash** [comment]
I'm closing this for now.  @pollei if you add tests please feel free to re-open this.

**Pull request closed without merging**